### ISSUE TITLE
Do an a11y pass

### ISSUE THREAD

**Issue by zkat** [BanchanArt/banchan#352]
I think we're probably doing a very very poor job of a11y, so we should do a pass and see where we're at and start fixing things.

**Comment by alextait1** [comment]
Hey! I'm happy to help with this :) What's the best way for you to digest findings? For example, there is a button on the homepage (alarm bell icon for notifications) that are missing an accessible name. This is what would be read by a screen reader or targeted by voice to text users. 

One approach (often a good one) is to go global and do a sweep of all graphical buttons and giving them a fix. The best fix is visible text near the icon that is either button text or uses aria-labelledby with an associated element.

**Comment by alextait1** [comment]
Is it helpful to just spit out issues like this over time? Or is there a more helpful way for you to receive them?

**Comment by zkat** [comment]
If there's many places where we should apply the same fix, I would prefer that to be a single issue with many sub-tasks (something like this):

Invisible text [explanation of the issue] [possible remedy, if you can].

Some locations:

* [ ] navbar notification icon
* [ ] Commission listing page comment box

I'll also apply an `a11y` tag to them as they come in (I'll make it after this comment).

Thank you so much for this! I'll do my best to extrapolate as things come in and help fill out your findings, so don't feel pressured to find every single corner case where a specific thing happens, since I can probably think of other places (and will edit the issue accordingly with the additional locations). I just don't generally know how to make things accessible or how to know if they are (and I feel really bad about it).

**Comment by zkat** [comment]
Update: Oh, apparently we already had the a11y label, and there's a couple of issues for it: https://github.com/BanchanArt/banchan/issues?q=is%3Aopen+is%3Aissue+label%3Aaccessibility

**Comment by zkat** [comment]
Closing this. We can do accessibility testing over time and fix things as they come up.
